packages/cdk/big-decimal/src/public_api.ts

```diff
@@ -0,0 +1,4 @@
+// based on package "js-big-decimal@1.3.4"
+// https://github.com/royNiladri/js-big-decimal
```

Comment:
move it to documentation (md file) instead place here

Reply:
good idea